hooks/useConfig.ts

```diff
@@ -6,10 +6,8 @@ export const useConfig = () => {
   if (Array.isArray(query.id)) {
     throw new Error('Too many ids have been specified');
   }
-  if (Number(query?.id) === DaoId.PoolTogether) {
+  if (!query.id || !(query.id in config)) {
     return config[DaoId.PoolTogether];
-  } else if (Number(query?.id) === DaoId.Gitcoin) {
-    return config[DaoId.Gitcoin];
   }
-  return config[DaoId.PoolTogether];
+  return config[query.id as DaoId];
```

Comment:
Related to the comments about whether to default to a non-existent dao.

Reply:
sorry can you elaborate?

Reply:
Sort of the same issue referenced here https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/124#discussion_r1406581312. If the `query.id` is 'example' should this return `undefined` or the default dao like PoolTogether.

Reply:
Oh nevermind read it wrong, it defaults under the first if block.